pkg/motor/query.go

```diff
@@ -105,3 +106,11 @@ func (mtr *motorNodeImpl) QueryWhereIsByCreator(req mt.QueryWhereIsByCreatorRequ
 		WhereIs: ptrArr,
 	}, nil
 }
+
+func (mtr *motorNodeImpl) QueryObject(cid string) (map[string]interface{}, error) {
+	sh := shell.NewShell(mtr.Cosmos.GetIPFSApiAddress())
```

Comment:
there is a instance of shell already scoped within the `Motor`.

Reply:
I did not notice an instance in Motor but I added a reference to `motorNodeImpl`.